doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,139 @@
+.. _gh_labels:
```

Comment:
can this document be integrated into the existing issues document as a sub-section?

Reply:
Today that mini-page content is a bit disconnected from this one (I guess you meant issues.rst ?)
You have a PR editing that one.. So after this is merged, feel free to take this content in if you feel it fits better there.